scripts/recover-sqlite-from-backups.sh

```
@@ -0,0 +1,239 @@
+    print("RECOMMENDED_BASENAME=" + best[3])
+PY
+}
+
+compare_remote() {
+  if [ -z "${HOST}" ] || [ "${HOST}" = "localhost" ]; then
+    echo "Нужен DEPLOY_HOST"
+    exit 1
+  fi
+  echo "=== Сравнение снимков на ${HOST}:${REMOTE_DB_DIR} (только *.bak*) ==="
+  # shellcheck disable=SC2029
+  _compare_py_body | ssh ${SSH_OPTS} "${HOST}" "export D=\"${REMOTE_DB_DIR}\"; python3"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)